sheets/1/exercise-3.tex

```diff
@@ -1,150 +1,170 @@
 % !TeX root = ./jvk-blatt1.tex
+\lstset{
+basicstyle=\small,
+morecomment=[l]{*}
+}
 \excercise{Syntax}
 
-In dieser Aufgabe geht es darum, die Syntax von Java zu verstehen.
-In jeder Teilaufgabe wird ein bestimmter Teil von Java erklärt und du sollst ein Beispiel aus Aufgabe 1 finden. In \textbf{[]} steht immer wie viele verschiedene Beispiele du finden solltest.
-
+\noindent
+In dieser Aufgabe zeigen wir euch die Syntax von Java.
+Die Syntax beschreibt, wie der Code geschrieben werden muss, sodass er von einem Computer verstanden wird.\\
+Es kommen jetzt viele Definitionen, ihr müsst euch aber keine Sorgen machen wenn ihr nicht alles direkt versteht.
+Nehmt diese Aufgabe als Referenz zum Nachschauen, wenn ihr euch später bei etwas nicht sicher seid.
 \begin{enumerate}
-    \item \textbf{[2]} Bevor wir uns mit dem Java Programmcode an sich beschäftigen, sollten wir noch Wissen was Kommentare sind.
-          Weil es vorkommt, dass wir Notizen oder Erklärungen in die Java-Datei schreiben wollen, welche für andere Menschen oder einem selbst bestimmt sind. Da der Computer diese Notizen nicht lesen soll, müssen wir sie in der Java Datei (also im Programmcode) als Kommentare markieren.
-          Für kurze Kommentare kann man \lstinline[breaklines=false]{//} verwenden. Damit markiert man den Rest der Zeile als Kommentar. Beachte, dass manche Programme, automatisch einen Zeilenumbruch einfügen, wenn die Zeile zu lang wird.
-          Das ist aber meistens nur Darstellung und gilt nicht als ende der Zeile.
-          Alternativ kann man den Kommentar auch mit \lstinline[breaklines=false]{/* */} umschließen. Dadurch kann man auch ein mehrzeiligen kommentar schreiben. Dabei ist es Konvention, jede Zeile mit einem \textcolor{javagreen}{\texttt{*}} zu beginnen. (wie in dem Beispiel unten zu sehen).
-          \begin{lstlisting}[title=\textbf{Kommentar Beispiel}]
-// Ein kurzer Kommentar 
-// Man kann übrigens am fehlen einer Zeilenummer gut erkennen, wenn ein automatischer, rein visueller Umbruch stattfindet.
-    
-class void not_even_Java //Auch an der grünen Färbung kann man in den meisten Programmen den Kommentar erkennen.
-//Und in dieser PDF wird auch ein Pfeil als Markierung für einen automatischen Zeilenumbruch verwendet.
-/* Allerdings sollte man das eher vermeiden.
- * Für tatsächliche mehrzeilige Kommentare werden diese Blöcke empfohlen
- * Auch hier kann natürlich automatisch umgebrochen werden.
+    \item \teamsaufgabe{Blatt 1: Aufgabe 3}
+    \item \textbf{Schreibt} zu jeder Teilaufgabe selbst zwei Beispiele in einem Editor. 
```

Comment:
Es war hauptsächlich Lernziel Syntax in vorhandenem code richtig zu erkennen. Deshalb eher sowas wie markiere in main.java alle vorkommen des Syntaxes (oder ähnliches).

Reply:
Wir würden das hauptsächlich durch quiizze im teams lösen. Das hat den Vorteil das wir mitbekommen wie gut sich die Leute schlagen